src/components/RoomMainUi.tsx

```diff
@@ -1,16 +1,55 @@
-// HOST ROOM
 'use client'
 
 import { socketRoomEmitter } from '@/services/socket'
-
-import StoryPoints from '@/components/StoryPoints'
+import { useSocketListener } from '@/services/socket'
+import { useState, useEffect } from 'react'
+import StoryPointsContainer from '@/components/StoryPointsContainer'
 import CardsContainer from '@/components/CardsContainer'
 
-export default function RoomMainUi() {
+export type RoomInfoData = {
+	roomId: string
+	userName: string
+}
+
+export default function RoomMainUi({ roomId, userName }: RoomInfoData) {
+	// const [selectedStoryPoint, setSelectedStoryPoint] = useState<string>('')
+
+	// const resetStoryPointBtns = useSocketListener('reset-points')
+
+	// useEffect(() => {
```

Comment:
suggestion: Can we delete this commented code?

Reply:
DONE